analysis/src/CompletionBackEnd.ml

```diff
@@ -2255,7 +2234,7 @@ let rec processCompletable ~debug ~full ~scope ~env ~pos ~forHover completable =
          user has already written. Just complete for the rest. *)
       let newText =
         c.name ^ " {\n"
-        ^ (cases
+        ^ (cases |> List.sort String.compare
```

Comment:
I can't really explain this one but at least sorting should make this deterministic.